### PR TITLE
feat(rm): pkg_resource

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

-

### Changed

- Removed pkg_resources

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b rm-pkg-dist
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/utils/files.py [hunk 1]
```diff
@@ -8,6 +8,10 @@
 LOG = logging.getLogger(__name__)
 
 
+def get_project_root_dir() -> Path:
+    return Path(__file__).parent.parent
```
Comment (ivadym) [inline cg/utils/files.py:12]: This is also an option, but I'm not sure which one is best in this context. I ended up using the `importlib.resources` module for the SWEDAC delivery report logo path

```suggestion
from importlib.resources import files

def get_project_root_dir() -> Path:
    return files("cg")
```

**Review by ivadym — APPROVED** [review]
👍

File: cg/utils/files.py (hunk 1 above)
Comment (henrikstranneheim) [reply cg/utils/files.py]: I think your is better as it avoids the dunders access. Updated. @Vince-janv Do you know a good way to test the invoice part?

**Comment by henrikstranneheim** [comment]
### Generating and rendering delivery report with SWEDAC logo, 👍 :

Screenshot 2024-08-15 at 11 24 28

**Review by ivadym — APPROVED** [review]
Nice 💯

**Comment by henrikstranneheim** [comment]
@Vince-janv Any easy way of testing the rendering of the invoice? Otherwise I think this is good to go!

**Comment by henrikstranneheim** [comment]
@Vince-janv Can I merge this next week? Should be fairly simple to fix the invoice rendering if there is a bug.

**Comment by Vince-janv** [comment]
@henrikstranneheim Sorry I missed the first message. I tested now and it seems to not be working. Here's a link to reproduce it in stage: https://cg-statusdb-stage.scilifelab.se/invoices/1626/excel?cost_center=KTH.

Here's the log I found on the stage VM:

> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]: 2024-09-02 07:57:42 5ed5b6aff961 cg.server.app[12] ERROR Exception on /invoices/1626/excel [GET]
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]: Traceback (most recent call last):
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 1455, in wsgi_app
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     response = self.full_dispatch_request()
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 869, in full_dispatch_request
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     rv = self.handle_user_exception(e)
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/usr/local/lib/python3.11/site-packages/flask_cors/extension.py", line 178, in wrapped_function
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     return cors_after_request(app.make_response(f(*args, **kwargs)))
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:                                                 ^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 867, in full_dispatch_request
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     rv = self.dispatch_request()
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:          ^^^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 852, in dispatch_request
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/home/src/app/cg/server/invoices/views.py", line 197, in invoice_template
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     workbook = render_xlsx(invoice_dict)
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:                ^^^^^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/home/src/app/cg/apps/invoice/render.py", line 46, in render_xlsx
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     workbook = load_workbook(template_path.as_posix())
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/usr/local/lib/python3.11/site-packages/openpyxl/reader/excel.py", line 315, in load_workbook
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     reader = ExcelReader(filename, read_only, keep_vba,
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/usr/local/lib/python3.11/site-packages/openpyxl/reader/excel.py", line 124, in __init__
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     self.archive = _validate_archive(fn)
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:                    ^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/usr/local/lib/python3.11/site-packages/openpyxl/reader/excel.py", line 96, in _validate_archive
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     archive = ZipFile(filename, 'r')
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:               ^^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:   File "/usr/local/lib/python3.11/zipfile.py", line 1294, in __init__
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:     self.fp = io.open(file, filemode)
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]:               ^^^^^^^^^^^^^^^^^^^^^^^
> Sep 02 07:57:42 cg-services-stage.scilifelab.se podman[2246673]: FileNotFoundError: [Errno 2] No such file or directory: '/home/src/app/cg/templates/KTH_sample_invoice.xlsx'

**Comment by henrikstranneheim** [comment]
@Vince-janv Thx for testing! Removed `cg` from the base. Could you test again?

**Comment by Vince-janv** [comment]
Still got an error but the path looks a bit different
`FileNotFoundError: [Errno 2] No such file or directory: '/home/src/app/templates/KTH_sample_invoice.xlsx'`

**Comment by henrikstranneheim** [comment]
@Vince-janv I think I got it this time. What do you think?

**Comment by Vince-janv** [comment]
Now it's working 🥳

**Pull request merged**